src/main/java/com/exactpro/th2/FixHandler.java

```diff
@@ -164,16 +164,22 @@ public FixHandler(IHandlerContext context) {
             msgSeqNum = new AtomicInteger(sequences.getClientSeq());
             serverMsgSeqNum = new AtomicInteger(sequences.getServerSeq());
         }
+
+        LOGGER.info("Initial sequences are: client - {}, server - {}", msgSeqNum.get(), serverMsgSeqNum.get());
+
         if(settings.getSessionStartTime() != null) {
             Objects.requireNonNull(settings.getSessionEndTime(), "Session end is required when session start is presented");
+            if(settings.getSessionStartTime().isBefore(settings.getSessionEndTime())) {
```

Comment:
> if(settings.getSessionStartTime().isBefore(settings.getSessionEndTime())) {

This'll have false positives for some cases (e.g. when start is at `08:00` and end is at `02:00` (of the next day))